xml/issue3403.xml

```diff
@@ -30,6 +30,63 @@ expression, I think it's an improvement?
 <p>Priority to 2 after reflector discussion.</p>
 
 <note>2020-07-22 Casey provides wording</note>
+
+<p><strong>Previous resolution [SUPERSEDED]:</strong></p>
+<blockquote class="note">
+<p>
+This wording is relative to <a href="http://wg21.link/n4861">N4861</a>.
+</p>
+
+<ol>
+<li><p>Add a new paragraph after paragraph 1 in <sref ref="[ranges.syn]"/>:</p>
+<blockquote>
+<ins>
+-?- Also within this clause,
```

Comment:
```suggestion
-?- Also within this Clause,
```

Reply:
Thanks - submitted https://github.com/cplusplus/draft/pull/4103 to fix the pre-existing occurrence in [ranges.syn]/1.